lib/dockerize_rails/docker_commands/docker_build.rb

```diff
@@ -2,10 +2,15 @@ module DockerizeRails
   module DockerCommands
     module DockerBuild
       def self.build_rails
+        if Docker::Image.exist? DockerHelpers.get_name(:rails, :image)
+          puts " ==> Image >#{DockerHelpers.get_name(:rails, :image)}< already exists".green
+          return 0
+        end
         build_docker_image(
-          "#{Constants::CONFIG_DIRECTORY_NAME}/#{Constants::RAILS_DIRECTORY_NAME}/Dockerfile",
-          "#{DRConfig.application_name}_rails"
+            "#{Constants::CONFIG_DIRECTORY_NAME}/#{Constants::RAILS_DIRECTORY_NAME}/Dockerfile",
```

Comment:
__[rubocop]__
* Indent the first parameter one step more than the start of the previous line.

<!-- Comment by [SideCI](https://sideci.com) -->